chunks/lang-rust/Dockerfile

```diff
@@ -18,17 +18,12 @@ RUN cp $HOME/.profile $HOME/.profile_orig \
     && rustup completions bash | sudo tee /etc/bash_completion.d/rustup.bash-completion > /dev/null \
     && rustup completions bash cargo | sudo tee /etc/bash_completion.d/rustup.cargo-bash-completion > /dev/null \
     && printf '%s\n' "$(grep -v -F -x -f $HOME/.profile_orig $HOME/.profile)" \
-                        'mkdir -m 0755 -p "${CARGO_HOME:-/workspace/.cargo}" 2>/dev/null' > $HOME/.bashrc.d/80-rust \
+                        'mkdir -m 0755 -p "${CARGO_HOME:-/workspace/.cargo}" 2>/dev/null' \
+                        'export CARGO_HOME=/workspace/.cargo' \
+                        'export PATH=$CARGO_HOME/bin:$PATH' > $HOME/.bashrc.d/80-rust \
     && _rustup_path="$(command -v rustup)" && mv "$_rustup_path" "${_rustup_path}.main" \
     && printf '%s\n' '#!/usr/bin/bash -eu' \
                         'exec env -u CARGO_HOME "$(command -v rustup.main)" "$@"' > "$_rustup_path" \
-    && chmod 0755 "$_rustup_path"
+    && chmod 0755 "$_rustup_path" && rm $HOME/.profile_orig
 
 RUN cargo install cargo-watch cargo-edit cargo-workspaces
-ENV CARGO_HOME=/workspace/.cargo
-ENV PATH=$CARGO_HOME/bin:$PATH
-
-RUN sudo mkdir -p $CARGO_HOME \
```

Comment:
Just double checking, is it intentional to remove `chown` here?

Reply:
Yes, I'm not sure how these lines got added by themselves again as I had removed them in a PR previously https://github.com/gitpod-io/workspace-images/pull/587/files